version.go

```diff
@@ -130,3 +132,13 @@ func (r *Range) Contains(v *Version) bool {
 	// Lower <= v && Upper > v
 	return r.Lower.Compare(v) != 1 && r.Upper.Compare(v) == 1
 }
+
+// FromSemver is the SemVer to claircore.Version mapping used by this package.
+func FromSemver(v *semver.Version) (out Version) {
```

Comment:
NAK on adding `semver` to the `claircore` API

Reply:
Reverted, lmk if you think of a decent common place to add it